lib/android/build.gradle

```diff
@@ -17,6 +17,7 @@ android {
 }
 
 dependencies {
+  def supportLibVersion = safeExtGet('supportLibVersion', '28.0.0')
```

Comment:
Work perfectly!

Reply:
You guys should properly on npm as this line is missing when run npm install react-native-maps

Reply:
working perfectly!

Reply:
This still happens for me on `"react-native": "0.61.5"` and `"react-native-maps": "0.26.1"`:

```
* Where:
Build file '.../node_modules/react-native-maps/lib/android/build.gradle' line: 20

* What went wrong:
A problem occurred evaluating project ':react-native-maps'.
> Could not get unknown property 'supportLibVersion' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

Reply:
Works fine! Any plan to release a patch version with this fix?